scripts/cron.lua

```diff
@@ -0,0 +1,41 @@
+function compute(process, message, opts)
+    -- Early return when no body is provided
+    -- This handles the initial invocation during process setup
+    if not message.body or not message.body.body then
+        return process
+    end
+    
+    -- Initialize crons table if it doesn't exist
+    process.crons = process.crons or {}
+    
+    local command = message.body.body.path
+    ao.event("debug_cron", { "compute command", command })
+    
+    if command == "put" then
+        -- Standard addition to the cache with task_id
+        ao.event("debug_cron", { "adding task", message.body.body.task_id })
+        table.insert(process.crons, message.body)
+        
+    elseif command == "remove" then
+        -- Remove an entry by task_id
+        local task_id = message.body.body.task_id
```

Comment:
may want to do an assert that `message.body` and `message.body.body` and `message.body.body.task_id` all exist